docs/content/en/docs/IOP/Deployment/Kubernetes/wopiserver.md

```diff
@@ -0,0 +1,184 @@
+---
+title: "Deploying and configuring the WOPI Server with Collabora Code"
+linkTitle: "WOPI Server"
+weight: 17
+description: >
+  Enable the deployment of the WOPI Server as part of the IOP.
+---
+
+## Prerequisites
+
+- This tutorial focuses on enabling the [`WOPI Server`](https://github.com/cs3org/wopiserver) as part of an **existing IOP deployment**. For detailed instructions on how to achieve the latter, [refer to the Kubernetes section of this documentation](https://developer.sciencemesh.io/docs/iop/deployment/kubernetes).
+
+> **Note:** Verify you have the `sciencemesh/iop` chart sources on version `0.0.3` or greater:
+
+```bash
+helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "sciencemesh" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+
+helm search repo sciencemesh/iop
+NAME                    	CHART VERSION	APP VERSION	DESCRIPTION
+sciencemesh/iop         	0.0.3        	0.0.1      	ScienceMesh IOP is the reference Federated Scie...
+```
+
+## (Optional) Deploying a Collabora Code instance on your cluster
+
+To demonstrate how an office-online application provider can integrate with the IOP through the WOPI protocol, we need an actual provider we can connect to. If you don't have such application at hand, don't worry. You can use a self-contained [Collabora Code](https://www.collaboraoffice.com/code/) installation on Kubernetes instead.
+
+The official Helm chart for [`stable/collabora-code`](https://hub.helm.sh/charts/stable/collabora-code) can be used to deploy everything on our cluster in a very simple way.
+
+First, we need to add the Chart's repo to our helm sources:
+
+```bash
+helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+"stable" has been added to your repositories
+
+helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Successfully got an update from the "sciencemesh" chart repository
+...Successfully got an update from the "stable" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+
+helm search repo stable/collabora-code
+NAME                 	CHART VERSION	APP VERSION	DESCRIPTION
+stable/collabora-code	1.0.6        	4.0.3.1    	A Helm chart for Collabora Office - CODE-Edition
```

Comment:
As just discovered, `collabora-code` must be version 4.2+

Reply:
Yep, I included a note pointing at this a few lines below on https://github.com/sciencemesh/sciencemesh/pull/142/commits/23f8b0edf6f15d343128272be4a2d3944008d6b1.